pkgs/applications/version-management/src/default.nix

```diff
@@ -1,25 +1,32 @@
-{ stdenv, fetchurl, python, rcs, git }:
+{ stdenv, fetchurl, python, rcs, git, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "src-1.11";
+  name = "src-${version}";
+  version = "1.12";
 
   src = fetchurl {
     url = "http://www.catb.org/~esr/src/${name}.tar.gz";
-    sha256 = "07kj0ri0s0vn8s54yvkyzaag332spxs0379r718b80y31c4mgbyl";
+    sha256 = "1m6rjbizx9win3jkciyx176sfy98r5arb1g3l6aqnqam9gpr44zm";
   };
 
-  buildInputs = [ python rcs git ];
+  buildInputs = [ python rcs git makeWrapper ];
 
   preConfigure = ''
     patchShebangs .
   '';
 
   makeFlags = [ "prefix=$(out)" ];
 
-  meta = {
+  postInstall = ''
+  wrapProgram $out/bin/src \
```

Comment:
By convention I usually see the contents of multiline strings indented another two spaces.